packages/postgres-database/src/simcore_postgres_database/models/workspaces.py

```diff
@@ -0,0 +1,76 @@
+import sqlalchemy as sa
+
+from ._common import column_created_datetime, column_modified_datetime
+from .base import metadata
+
+workspaces = sa.Table(
+    "workspaces",
+    metadata,
+    sa.Column(
+        "workspace_id",
```

Comment:
MINOR: Again about conventions. Why we are not having just `id` here?

Reply:
I just prefer it like this, is there an agreed convention?

Reply:
discussed, keeping `workspace_id`